libs/ui/layout/src/lib/figure/figure.component.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  Component,
+  OnInit,
+  ChangeDetectionStrategy,
+  Input,
+} from '@angular/core'
+
+@Component({
+  selector: 'gn-ui-figure',
+  templateUrl: './figure.component.html',
+  styleUrls: ['./figure.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class FigureComponent {
+  @Input() icon: string
+  @Input() title: string
+  @Input() figure: string
```

Comment:
I wonder why you switched back to string for the actual figure? Having a number means it can be formatted correctly for the user locale, and also shown in different ways depending on available space, e.g. : the same data could be shown as `12k`, `12,127`, `12,127.84` depending on the width of the component

Having figures that are strings makes sense as well, for qualitative data for example, eg. : "Most common name in 2021: **John**", but this is a less common use case I think.